ambry-cloud/src/main/java/com.github.ambry.cloud/CloudStorageManager.java

```diff
@@ -103,8 +106,7 @@ public boolean controlCompactionForBlobStore(PartitionId id, boolean enabled) {
 
   @Override
   public boolean removeBlobStore(PartitionId id) {
-    partitionTostore.remove(id);
-    return true;
+    return partitionTostore.remove(id) == null ? false : true;
```

Comment:
Simpler: return partitionTostore.remove(id) != null;
Also, should be partitionToStore (camel case)

Reply:
Never mind, already fixed.